app/views/entities/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= render "shared/back" %>
+<div class="transact-new">
+    <h3>Add new Transactions</h3>
+
+    <%= form_with(model: @entity, url: category_entities_path, method: :post, class: 'create-new-transact-form') do |f| %>
+
+        <div class="field">
+            <%= f.label :name %> <br>
+            <%= f.text_field :name, placeholder: 'Enter Transaction name', class: 'name-input' %>
+        </div>
+
+        <div class="field">
+            <%= f.label :amount %><br>
+            <%= f.number_field :amount, step: 1, placeholder: 'How much did you spend', class: 'amount-input' %>
+        </div>
+
+        <% if @categories %>
+        <div class="cat-select-field">
+            <%= f.label :categories %>
+            <%= f.collection_check_boxes(:category_ids, @categories, :id, :name) do |b| %>
+                <div class="checkbox">
+                <%= b.check_box + b.label %>
+                </div>
+            <% end %>
+        </div>
+        <% end %>
+            <%= f.submit 'Save', class: 'Submit-transaction-btn' %>
+
+    <% end %>
+</div>
```

Comment:
- [ ] It will be better to follow the design guidelines in Behance design by using the same style for inputs and for the save button use the green button
> Input design 
![image](https://github.com/bill7pearl/BudgetWise/assets/53402600/4db1d57d-9ba7-4166-9ac5-8c30daa8f788)
> Button design 
![image](https://github.com/bill7pearl/BudgetWise/assets/53402600/d458efd0-10b0-4731-881e-2de66e3b0c34)

Reply:
done, I have matched the buttons design